src/main/java/dinkplugin/notifiers/data/PkNotificationData.java

```diff
@@ -0,0 +1,47 @@
+package dinkplugin.notifiers.data;
+
+import dinkplugin.message.Field;
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import net.runelite.api.coords.WorldPoint;
+import net.runelite.api.kit.KitType;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class PkNotificationData extends NotificationData {
+
+    String name;
+    int combatLevel;
+    Map<KitType, SerializedItemStack> equipment;
+    int world;
+    WorldPoint location;
+    int myHitpoints;
+    int myLastDamage;
```

Comment:
Should any of these variables be available for the string template replacement (e.g., `%LEVEL%`)?

Reply:
I think we can skip that until a request for it comes through